### PR TITLE
Bump SDK wheels to 2023.1.0 (=1.0.0.2)

### PR DESCRIPTION
**Pull request by marcelveldt**
Bump the SDK wheels to 2023.1.0 (built yesterday) which is based on official 1.0.0.2 patch release of the SDK

### CODE REVIEW

**Comment by marcelveldt** [comment]
not ready for primetime yet, there are breaking changes in the cluster models

**Comment by marcelveldt** [comment]
Included a lean and mean "fix" for the renamed Basic cluster.
I will fix this for real later when I'm working on the data structure where we build a mapping up front that maps cluster ID --> Class instead of the auto inspecting hocus pocus

File: matter_server/common/helpers/util.py [hunk 1]
```diff
@@ -53,6 +53,10 @@
 CHIP_CLUSTERS_PKG_NAME = "home-assistant-chip-clusters"
 CHIP_CORE_PKG_NAME = "home-assistant-chip-core"
 
+# TEMP: Basic Cluster got renamed in SDK version v1.0.0.2
+# we will fix this later when we're basic our datastructure
```
Comment (MartinHjelmare) [inline matter_server/common/helpers/util.py:57]:
```suggestion
# we will fix this later when we're basing our datastructure
```

File: matter_server/common/helpers/util.py [hunk 2]
```diff
@@ -53,6 +53,10 @@
...
+# on the id's instead of types.
```
Comment (MartinHjelmare) [inline matter_server/common/helpers/util.py:58]:
```suggestion
# on the ids instead of types.
```

File: matter_server/common/helpers/util.py [hunk 3]
```diff
@@ -53,6 +53,10 @@
...
+chip.clusters.Objects.Basic = chip.clusters.Objects.BasicInformation
```
Comment (MartinHjelmare) [inline matter_server/common/helpers/util.py:59]:
```suggestion
chip.clusters.Objects.Basic = chip.clusters.Objects.BasicInformation

```

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**